### ISSUE TITLE
FSqrt returns two valid responses for one input.

### ISSUE THREAD

**Issue by ccelio** [ucb-bar/berkeley-hardfloat#15]
**TL;DR:** I am sending one `fsqrt` request to `hardfloat.DivSqrtRecF64`. I get back one response the cycle after and then a second response back ~20 cycles later.

Waveform below.

----

I am using commit (f38b8beeb3fbce40ee3d9c1e132c8f60ed05f4aa) which is what rocket-chip master uses (as well as the boom-devel branch of rocket-chip).

------
**Questions:**

* Am I using `DivSqrtRecF64` incorrectly?
* Is this fixed from more recent refactorings of `DivSqrtRecF64`?

---------

<img width="1232" alt="screenshot 2017-03-25 at 11 53 32 pm" src="https://cloud.githubusercontent.com/assets/788854/24329274/4f979dca-11b6-11e7-8c1c-369284982ddc.png">

**Comment by ccelio** [comment]
I have set `io.sqrtOp` high because I assumed it was a `DontCare` if `inValid` is low. 

This particular error test-case would not trigger, but I'm not sure if that's just masking the issue.

**Comment by aswaterman** [comment]
Your assumptions about the interface seem to me to be correct.

It's impossible to tell where the bug is without the full waveform.  It looks like there are other operations in flight before the operation in question, so is it not possible that the first response is to a previous request, rather than to the operation in question?

**Comment by ccelio** [comment]
Okay, looks like the problem is on my end. 

I had counted requests and responses by eye and thought they were equal, but on closer inspection one request is held high waiting for ready to go high, but somehow misses a cycle of ready and stays high too long.

**Comment by aswaterman** [comment]
cool, thanks for sorting it out

**Comment by ccelio** [comment]
And thanks for the sanity check.
